t/rose-task-run/04-run-path-empty/suite.rc

```diff
@@ -24,8 +24,8 @@ install => my_task_1
            submission timeout = PT1M
            execution timeout  = PT1M
     [[install]]
-        command scripting = """rose task-run"""
+        script = """rose task-run"""
```

Comment:
I think we should do this as a separate pull request. We do have a lot of unnecessary quoted values in our test battery suites.